dataland-backend/src/test/kotlin/org/dataland/datalandbackend/service/DataManagerTest.kt

```diff
@@ -16,42 +19,57 @@ class DataManagerTest(
 
     val testManager = DataManager(edcClient, objectMapper)
 
-    val testCompanyNamesToStore = listOf("Imaginary-Company_I", "Fantasy-Company_II", "Dream-Company_III")
+    val testCompanyList = listOf(
+        CompanyInformation(
+            companyName = "Test-Company_1",
+            headquarters = "Test-Headquarters_1",
+            industrialSector = "Test-IndustrialSector_1",
+            marketCap = BigDecimal(100),
+            reportingDateOfMarketCap = LocalDate.now()
+        ),
+        CompanyInformation(
+            companyName = "Test-Company_2",
+            headquarters = "Test-Headquarters_2",
+            industrialSector = "Test-IndustrialSector_2",
+            marketCap = BigDecimal(200),
+            reportingDateOfMarketCap = LocalDate.now()
+        )
+    )
 
     @Test
     fun `add the first company and check if its name is as expected by using the return value of addCompany`() {
-        val companyMetaInformation = testManager.addCompany(testCompanyNamesToStore[0])
+        val companyMetaInformation = testManager.addCompany(testCompanyList[0])
         assertEquals(
-            companyMetaInformation.companyName, testCompanyNamesToStore[0],
+            companyMetaInformation.companyInformation.companyName, testCompanyList[0].companyName,
```

Comment:
Hier wird ja explizit nur auf den Namen geprüft. Zuvor war dies ja auch der einzige Wert der in der Companyinformation enthalten war, jetzt gibt es aber eine ganze Reihe von Feldern. Sollte hier nicht auch entsprechend auf alle Attribute bzw. das das erhaltene Objekt mit dem hochgeladenen konsistent ist geprüft werden?

Reply:
_Beginne Problem anzuschauen und evtl. zu fixen_

Reply:
Ich habe die Tests jetzt so verändert, dass abgleichen, ob die gespeicherte `StoredCompany` gleich der retrieveten ist.
Dadurch sollte gewährleistet sein, dass alles abgeglichen wird.